scheduler/src/cook/mesos/api.clj

```diff
@@ -729,7 +729,7 @@
         application (:job/application job)
         expected-runtime (:job/expected-runtime job)
         job-map {:command (:job/command job)
-                 :uuid (str (:job/uuid job))
+                 :uuid (:job/uuid job)
```

Comment:
The schema specifies a UUID, not a string. I am guessing there is some magic in compojure that cleans this up but I'd prefer it to conform to the schema here.